tests/NRedisStack.Tests/Search/SearchTests.cs

```diff
@@ -642,6 +642,10 @@ public void AlterAdd()
         Schema sc = new Schema().AddTextField("title", 1.0);
 
         Assert.True(ft.Create(index, FTCreateParams.CreateParams(), sc));
+
+        //sleep:
+        System.Threading.Thread.Sleep(2000);
```

Comment:
Why do you need this?

Reply:
Sometimes in the CI ft.Create command takes time and it causes errors in the tests, sleep makes sure that the command has finished running before the assertions